src/main/resources/apoc-config.xml

```diff
@@ -6,6 +6,7 @@
     <system/>
 <!--    <properties fileName="${env:HOME}/apoc.conf" config-optional="true"/>-->
     <properties fileName="${sys:NEO4J_CONF}/apoc.conf" config-optional="true"/> <!-- we're setting NEO4J_CONF in ApocConfig.java -->
+    <properties fileName="${sys:NEO4J_CONF}/neo4j.conf" config-optional="true"/> <!-- Make sure all properties from neo4j.conf get added in -->
 
```

Comment:
@bradnussbaum 
This fixes the issue where our broker configurations in the `neo4j.conf` file were being ignored. 

Adding this property ensures that all configurations in `neo4j.conf` (even those not preemptively declared) can still be used. Not sure if this is our long-term solution here, but so far this is the cleanest way I found to get the configuration to include our broker properties.